src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeIndustrialGreenhouse.java

```diff
@@ -870,18 +870,25 @@ protected String generateCurrentRecipeInfoString() {
             .append("%)\n");
 
         for (Map.Entry<ItemStack, Double> drop : GUIDropProgress.entrySet()) {
-            ret.append(
-                drop.getKey()
-                    .getDisplayName())
-                .append(": ")
+            int outputSize = Arrays.stream(mOutputItems)
+                .filter(s -> s.isItemEqual(drop.getKey()))
+                .mapToInt(i -> i.stackSize)
+                .sum();
+            if (outputSize == 0) continue;
```

Comment:
These overrides are made specifically to show fractional progression of the internal output buffer. (they are stored in `drop.getValue()`)
Please don't remove them.

Reply:
The information is debatable, if the greenhouse gets more than 1 product every 5 seconds, it's useless. If it takes more time to grow, then it is useful, but who will grow something in such slow volumes? I think it will be greatly accelerated through the number of seeds, fertilizers, etc.

For bees it is rather useless, it gives a little more profit, but it is not very noticeable. There is a stable production without this progress information.

Perhaps we should display this information only for those crops that have not matured in 5 seconds?

Reply:
> but who will grow something in such slow volumes

Primordial pearl berry :/